core/src/main/java/io/keen/client/java/http/Request.java

```diff
@@ -18,19 +18,19 @@
     public final String authorization;
     public final OutputSource body;
     public final Proxy proxy;
+    public final int connectTimeout;
+    public final int readTimeout;
 
-    ///// PUBLIC CONSTRUCTORS /////
+    ///// PUBLIC CONSTRUCTOR /////
 
-    public Request(URL url, String method, String authorization, OutputSource body) {
-        this(url, method, authorization, body, null);
-    }
-
-    public Request(URL url, String method, String authorization, OutputSource body, Proxy proxy) {
+    public Request(URL url, String method, String authorization, OutputSource body, Proxy proxy, int connectTimeout, int readTimeout) {
```

Comment:
Concern: Removing a public constructor is potentially a breaking change, isn't it? That means it's not safe to do in a minor version bump.

Reply:
yeap, I agree. Reverted it and marked as deprecated